course6/toyapp_alexs/displaylib/transformation3d.cc

```diff
@@ -0,0 +1,75 @@
+#include <cmath>
+
+#include "vector3d.h"
+#include "transformation3d.h"
+
+namespace Display {
+
+Move3D::Move3D(Vector3D move) : Transformation3D()
```

Comment:
"move" is a confusing name. Consider "translation"

Reply:
Well, it moves the object. Translation sounds confusing to me, let's discuss during the meeting.